app/views/admin/participants/_details.html.erb

```diff
@@ -1,58 +1,78 @@
-          Change <span class="govuk-visually-hidden">name</span>
-        <% end %>
-      </td>
-    </tr>
-    <tr class="govuk-table__row">
-      <th scope="row" class="govuk-table__header">Email address</th>
-      <td class="govuk-table__cell"><%= latest_induction_record.user.email %></td>
-      <td class="govuk-table__cell">
-        <%= govuk_link_to edit_email_admin_participant_path(latest_induction_record.participant_profile) do %>
-          Change <span class="govuk-visually-hidden">email</span>
-        <% end %>
-      </td>
-    </tr>
-    <tr class="govuk-table__row">
-      <th scope="row" class="govuk-table__header">ID</th>
-      <td class="govuk-table__cell"><%= latest_induction_record.user.id %></td>
-      <td class="govuk-table__cell"></td>
-    </tr>
-    <tr class="govuk-table__row">
-      <th scope="row" class="govuk-table__header">External ID</th>
-      <td class="govuk-table__cell"><%= latest_induction_record.participant_profile.participant_identity.external_identifier %></td>
-      <td class="govuk-table__cell"></td>
-    </tr>
-    <tr class="govuk-table__row">
-      <th scope="row" class="govuk-table__header">Associated email addresses</th>
-      <td class="govuk-table__cell">
-      <ul>
-        <% all_emails_associated_with_a_user(latest_induction_record).each do |email| %>
-         <li><%= email %></li>
-        <% end %>
-      </ul>
-      </td>
-      <td class="govuk-table__cell"></td>
-    </tr>
-    <tr class="govuk-table__row">
-      <th scope="row" class="govuk-table__header">Validation status</th>
-      <td class="govuk-table__cell"><%= render ParticipantStatusTagComponent.new(profile: latest_induction_record.participant_profile, induction_record: latest_induction_record) %></td>
-      <td class="govuk-table__cell"></td>
-    </tr>
-    <tr class="govuk-table__row">
-      <th scope="row" class="govuk-table__header">Notes</th>
-       <td class="govuk-table__cell"><%= latest_induction_record.participant_profile.notes? ? simple_format(latest_induction_record.participant_profile.notes, class: "govuk-body") : "No notes" %></td>
-
-       <td class="govuk-table__cell">
-         <%= govuk_link_to edit_admin_note_path(latest_induction_record.participant_profile) do %>
-           <span><%= latest_induction_record.participant_profile.notes.present? ? "Change notes" : "Add notes" %></span>
-         <% end %>
-       </td>
-    </tr>
-  </tbody>
-</table>
+<%= govuk_summary_list do |sl|
+  participant_name = latest_induction_record.user.full_name
+
+  sl.row do |row|
+    row.key(text: "Full name")
+    row.value(text: participant_name)
+    row.action(
+      href: edit_name_admin_participant_path(latest_induction_record.participant_profile),
+      visually_hidden_text: "name"
+    )
+  end
+
+  sl.row do |row|
+    row.key(text: "Email address")
+    row.value(text: latest_induction_record.user.email)
```

Comment:
Should we display the `latest_induction_record.preferred_identity.email` instead?
We also have `latest_induction_record.participant_profile.participant_identity.email`. 
One more thing that makes me suspect that ParticipantProfile and InductionRecord should be a single entity.

Reply:
This is a good question, I didn't realise we did  have a `preferred_identity` relationship. I think we should show that one too but this also has an edit action that updates the email address on the User model, so I think it should remain too (I suspect it's the most useful one for support)